### PR TITLE
Storage/lf related ids

### PR DESCRIPTION
**Pull request by lucyrfox**
## Summary
* what changes have you made?
* Added /relatedIdExists get method which takes in a related id and checks whether or not an entry with the exists in the MediaMentions table.
* Modified default value of relatedMentionId such that if the field is not provided, the value is NULL.
* Created default constructor for SQLCommandController to provide default connection information.

## Testing Plan
* what have you done to test these changes?
* Tested in Postman (Storage Util / Related ID Exists). Tested both IDs that exist in MediaMentions table and that do not.

## Affected Services
* which services are your changes affecting (ex: data collection, etc.)
* Storage

### CODE REVIEW

File: Storage/src/main/java/storage/controllers/SQLCommandController.java [hunk 1]
```diff
@@ -24,6 +23,19 @@
     Connection conn;
     Statement stmt;
 
+    /**
+     * If no parameters are passed in, default connection is made.
+     * @throws SQLException
+     */
+    public SQLCommandController() throws SQLException {
+        this.connectionstringurl = "jdbc:mysql://mysql_db:3306/dateline2.0";
```
Comment (spencerc99) [inline Storage/src/main/java/storage/controllers/SQLCommandController.java]: this shouldnt block this from merging in but we should put this stuff in a config/environment file and read it in somehow instead of just hardcoded

File: Storage/src/main/java/storage/models/MediaMention.java [hunk 2]
```diff
@@ -13,7 +13,7 @@ public MediaMention(){}
     public MediaMention(
         String url,
         String shortUrl,
-        long relatedMentionId,
+        Long relatedMentionId,
```
Comment (spencerc99) [inline Storage/src/main/java/storage/models/MediaMention.java]: why was this change needed just curious?

Comment (lucyrfox) [reply Storage/src/main/java/storage/models/MediaMention.java]: A long can’t be null bc it’s primitive - so when relatedMentionId was not provided in json input, it was being set to 0 by default instead of null. Changing to Long allows null

File: Storage/src/main/java/storage/dbclients/MediaMentionsClient.java [hunk 3]
```diff
@@ -131,4 +128,8 @@ public String delete(long id) {
     String sqlDelete = "delete from MediaMentions where mediaMentionId=" + id;
     return sqlDelete;
   }
+
+  public String checkRelatedID(long id) {
+    return "SELECT COUNT(*) FROM MediaMentions WHERE relatedMentionId = " + id + " LIMIT 1;";
```
[the remark this answers is not in the document]
Comment (lucyrfox) [reply Storage/src/main/java/storage/dbclients/MediaMentionsClient.java]: For efficiency — this tells COUNT to stop looking as soon as a row is found (—> just returns 0 or 1). Without LIMIT 1 it would look through the whole table every time and return total number of matching rows

[the remark this answers is not in the document]
Comment (willkoh) [reply Storage/src/main/java/storage/dbclients/MediaMentionsClient.java]: Oh gotcha so it's like a bool value that we want. that makes sense. Thanks!
